src/components/ErrorBoundary.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import postErrorToSlack from 'utils/postErrorToSlack';
+import { connect } from 'react-redux';
+import Error from 'pages/_error';
+
+class ErrorBoundary extends React.Component {
```

Comment:
now that we know that the bundle size increase comes from the crypto libraries, do you think it's worth using our custom component instead of the library? do you expect much overhead having to maintain it? do you think there are going to be new features in the library that could be beneficial?

Reply:
Hmm... I don't expect much changes to this component. The only place where I foresee modification is inside the "componentDidCatch" block, which is the handler for the raised error. Besides that, this component basically acts like a try-catch block.

Under the hood, the library is also actually using the same code, but just abstracting the functionality into props.

The equivalent piece of code in this component vs that provided by the library are : 
- Display an error message (`FallbackComponent` in the library's API)
This is implemented by the if statement in the `render()` method. Equivalent to [https://github.com/bvaughn/react-error-boundary/blob/master/src/index.tsx#L129-L130](https://github.com/bvaughn/react-error-boundary/blob/master/src/index.tsx#L129-L130)

- Catch and act on erorr (`onError` in the library's API)
This is implemented by the code block under `componentDidCatch`. Equivalent to [https://github.com/bvaughn/react-error-boundary/blob/master/src/index.tsx#L87-L89](https://github.com/bvaughn/react-error-boundary/blob/master/src/index.tsx#L87-L89)

I don't think the other APIs provided by the library (even `useErrorHandler`) will be used in our use case.

Therefore, I'd say that this custom component suits our use case, with less baggage.